src/Main.java

```diff
@@ -2,6 +2,79 @@
 // click the <icon src="AllIcons.Actions.Execute"/> icon in the gutter.
 public class Main {
     public static void main(String[] args) {
-        System.out.printf("Hello and welcome!");
+        //Task1
+        int age = 16;
+        if (age >= 18) {
+            System.out.println("Если возраст человека равен " + age + ", то он совершеннолетний.");
+        } else {
+            System.out.println("Если возраст человека равен " + age + ", он не достиг совершеннолетия, нужно немного подождать.");
+        }
+        //Task2
+        int temperature = 35;
+        if (temperature <= 5) {
+            System.out.println("На улице " + temperature + " градусов, нужно надеть шапку.");
+        } else {
+            System.out.println("На улице " + temperature + " градусов, можно идти без шапки.");
+        }
+        //Task3
+        int speedOfCar = 75;
+        if (speedOfCar >= 60) {
+            System.out.println("Если скорость " + speedOfCar + "км/ч, то придется заплатить штраф");
+        } else {
+            System.out.println("Если скорость " + speedOfCar + "км/ч, можно ездить спокойно");
+        }
+        //Task4
+        int ageOfHuman = 2;
+        if (ageOfHuman >= 2 && ageOfHuman <= 6) {
+            System.out.println("Если возраст человека равен " + ageOfHuman + ", то ему нужно ходить в детский сад");
+        }
+        if (ageOfHuman >= 7 && ageOfHuman <= 17) {
+            System.out.println("Если возраст человека равен " + ageOfHuman + ", то ему нужно ходить в школу");
+        }
+        if (ageOfHuman >= 18 && ageOfHuman <= 24) {
+            System.out.println("Если возраст человека равен " + ageOfHuman + ", то его место в университете");
+        }
+        if (ageOfHuman > 24) {
+            System.out.println("Если возраст человека равен " + ageOfHuman + ", то eму пора ходить на работу");
+        }
+        //Task5
+        int ageOfChildren = 2;
```

Comment:
можно было не создавать новую переменную, а просто использовать уже созданную
```suggestion
        age = 2;
```